lib/core/form/services/widget-visibility.service.spec.ts

```diff
@@ -916,10 +906,10 @@ describe('WidgetVisibilityService', () => {
         });
 
         it('should not set null value when the field is not visibile', () => {
-            visibilityObjTest.leftFormFieldId = 'test_4';
+            visibilityObjTest.leftFormFieldId = 'FIELD_TEST';
             visibilityObjTest.operator = '!=';
-            visibilityObjTest.rightFormFieldId = 'dropdown';
-            const fakeFormField: FormFieldModel = new FormFieldModel(formTest, jsonFieldFake);
+            visibilityObjTest.rightFormFieldId = 'RIGHT_FORM_FIELD_ID';
+            const fakeFormField: FormFieldModel = new FormFieldModel(fakeFormWithField, jsonFieldFake);
```

Comment:
Replacing the mock here and in any other places will not fix the unit tests.

Reply:
The existing formModel (formTest) hold only the _values_ property, and does not contain the  _fields_ property in the model. In this update, we check the _isValid_ property of the fields, which will not be available in the currently used formModel (formTest), which causes the tests to fail. Hence the changes.